Misc/NEWS.d/next/Core_and_Builtins/2025-01-10-18-56-20.gh-issue-128563.baDvls.rst

```diff
@@ -0,0 +1 @@
+A new type of interpreter has been added to CPython. This interpreter uses tail calls for its instruction handlers. Preliminary benchmark results suggest 7-11% geometric mean faster on pyperformance (depending on platform), and up to 30% faster on Python-intensive workloads. This interpreter currently only works on newer compilers, such as ``clang-19``. Other compilers will continue using the old interpreter. Patch by Ken Jin, with ideas by Mark Shannon, Garret Gu, Haoran Xu, and Josh Haberman.
```

Comment:
Using tail calls as a dispatching mechanism is an old idea. It possibly dates back to the original threaded code implementation back in the 1960s? Best mention that our idea is just to apply it to CPython.

Reply:
I said that our idea is implementing it in CPython.